engine/memory.asm

```diff
@@ -55,13 +55,14 @@ VH_DSKROM       EQU     4                       * Disk BASIC code image (used wh
 VH_HIGHROM      EQU     5                       * Top 8k of cartridge ROM space (used with CoCoNET MicroSD Pak)
 
 VH_SPRERASE     EQU     6                       * Sprite background pixels
-VH_LVLOBJCODE   EQU     7                       * Level and object handling code page
-VH_BKTILES      EQU     8                       * background block texture data (max 8 8k pages)
-VH_BKMAP        EQU     16                      * background tilemap (max 8 8k pages)
-VH_ZIPDATA      EQU     24                      * decompressor tables and large state data (one page)
-VH_ZIPBUF       EQU     25                      * decompressed data buffer (max 5 8k pages)
-VH_SOUNDDATA    EQU     30                      * audio waveform pages (max 8 8k pages)
-VH_SPRCODE      EQU     38                      * sprite draw/erase code pages
+VH_LVLOBJCODE1  EQU     7                       * First level and object handling code page
```

Comment:
I sort of don't like the virtual handle values all being dependent upon OBJPAGES after #7.  Can we move this block of pages (VH_LVLOBJCODE1-VH_LVLOBJCODEX) down the list right before the VH_SPRCODE block at the end?

Reply:
Fixed